MixStaticSeq/task.py

```diff
@@ -94,6 +108,8 @@ def main(args):
     parser.add_argument("--mix_hidden_dim", default=64, type=int, help='fully connected layer size for mix model')
     parser.add_argument("--nonseq_representation_dim", default=64, type=int,
                         help='representation dim for nonseq features')
+    parser.add_argument("--log_gradients", action='store_true',
```

Comment:
no default value?

Reply:
action='store_true' means default is false